frontend/src/components/Share/Share.js

```diff
@@ -2,20 +2,99 @@ import React from 'react';
 import './Share.scss';
 import share from "../../assets/share.svg";
 
+class Share extends React.Component {
```

Comment:
Maybe you can convert it to a functional component? Class components are fine, but with functional ones we have less overload. have a look at https://reactjs.org/docs/components-and-props.html

Reply:
And also look at other components within this project, if you need references. Only a few should be `class`